src/config.ts

```diff
@@ -330,6 +331,9 @@ export const TRANSIFEX_TOKEN = "1/9ac6d0a1efcda679e72e470221e71f4b0497f7ab";
 export async function initializeNode(): Promise<NodeInfo> {
   console.log("config initialize called");
   const nodeList = await NodeList();
+  if (nodeList.length < 1) {
+    throw new InitializeNodeFailedError();
+  }
   nodeList.sort((a, b) => {
     return a.clientCount - b.clientCount;
   });
```

Comment:
Not really related to this PR per se, but do we need to sort this list if we're going to pick randomly?

Reply:
yes, this is temporary loadbalance in client side before rpc node support alb(or nlb).